apps/api/prisma/migrations/20260720000000_hot_path_indexes/migration.sql

```
@@ -0,0 +1,20 @@
+-- Performance: cover the hottest write/read paths with indexes that actually match
+-- their query shape.
+--
+-- WatchEvent: `recordWatchEvent` (the dedup check run on every scrobble/webhook watch
+-- event) filters on profileId + type + imdbId (+ season/episode for episodes) — the
+-- old profileId-only index left Postgres scanning every event for the profile. The
+-- new composite index also covers /watch/history and /watch/stats/* ordering/filtering
+-- by profileId + watchedAt, so the old plain profileId index is now redundant (it's a
+-- prefix of the new one) and is dropped to avoid the extra write-time index maintenance.
+DROP INDEX "WatchEvent_profileId_idx";
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)